src/test/java/com/training/assignment/tests/AssignmentTest.java

```diff
@@ -80,4 +82,33 @@ public void testOperationString() {
         softAssert.assertThat(assignment.getIndex(new String[]{"Paulo Dybala", "Federico", "Gianluigi", "Ronaldo", "Messi"}, "Ronaldo"))
                 .as("Ronaldo string not found in list").isEqualTo(3);
     }
+
+    @Test
+    public void testOperationWithLoops() {
+        softAssert.assertThat(operation.getCompanyReviews("Google")).
+                as("review not found").isEqualTo("Best platform where u will get solution for everything");
```

Comment:
in as(), the description should be like , Review of google is not matching with best platform where u will get solution for everything